React/Views/RCTView.m

```diff
@@ -779,6 +779,9 @@ - (void)viewDidMoveToWindow
                                                  name:NSViewBoundsDidChangeNotification
                                                object:[[self enclosingScrollView] contentView]];
   }
+
+  [self reactViewDidMoveToWindow]; // TODO(macOS)
```

Comment:
Doesn't need a tag since this is already in a giant macOS block. But another thought..

I think I'd prefer you keep the tag, adding the reference to a new issue explaining the bug. That will make it easier to upstream this particular change, which feels worthy to go into RN Core.